tests/drivers/i2c/i2c_slave_api/boards/nucleo_f091rc.conf

```diff
@@ -2,3 +2,4 @@ CONFIG_I2C_STM32_V2=y
 CONFIG_I2C_STM32_INTERRUPT=y
 CONFIG_I2C_1=y
 CONFIG_I2C_2=y
+CONFIG_I2C_VIRTUAL=n
```

Comment:
unrelated change

Reply:
Actually, it is related.

Unlike all other samples, i2c_slave_api had two different files for the "common" configuration.

One that was used when no board configuration was needed (prj_virtual.conf), and another that is combined with board config (prj_base.conf).

This pattern is not supported by boilerplate.cmake, so this sample was changed to only have one "common" config file (prj.conf). Doing so required modifying the board configuration files to disable some of the "common" configuration from prj_base.conf.

Reply:
Ok.